### ISSUE TITLE
Zeronet doesn't works no more

### ISSUE THREAD

**Issue by LopezGorra** [HelloZeroNet/ZeroNet#2739]
Error: Se perdió la conexión con UiServer Websocket. Reconectando. (The connection to the UiServer Websocket was lost. Reconnecting. )

  * ZeroNet version: last
  * Operating system: linux debian x64 bits
  * Web browser: firefox
  * Tor status: disabled
  * Opened port: yes/no
  * Special configuration: ____
  * 
![Screenshot_2021-03-05_16:43:27](https://user-images.githubusercontent.com/80126394/110165811-3a522c00-7dd2-11eb-8fc8-cce2324277be.png)

**Comment by krzotr** [comment]
Please check the log file of the application. Everything should be there.

**Comment by slrslr** [comment]
yes find log directory and latest log file in it check normal and error log, you may try to launch ZN from within command line:
cd /path/to/zeronet
./ZeroNet.sh --debug --console_log_level DEBUG
(maybe ZeroNet.sh not found, so check which other name to use. if .py extension, then prefix the command by python)
You can try to backup zeronet directory contents except /data which may be too big) and replace the directory contents with the files downloaded from https://github.com/HelloZeroNet/ZeroNet/releases or from zeronet.io
